docs/included-feature-flags.md

```diff
@@ -20,4 +20,8 @@ message will be displayed when the admin level is changed.
 
 ## AllowChatCommandExecuting (default: `true`)
 If true, commands can be executed from the chat with
-prefixes used. If false, chat executing will be ignored.
\ No newline at end of file
+prefixes used. If false, chat executing will be ignored.
+
+## AllowDroppingSwords (default: `false`)
+If true, players will be able to drop swords. 
+If false they won't.
```

Comment:
Can a comma be added here to be consistent?

Reply:
Done.